db/migrate/20251024143311_remove_vso_grp_index_and_add_unique_index.rb

```diff
@@ -0,0 +1,39 @@
+class RemoveVSOGrpIndexAndAddUniqueIndex < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  class Rep < ActiveRecord::Base
+    self.table_name = 'veteran_representatives'
+  end
+
+  def up
+    remove_index :veteran_representatives,
+                 name: :index_vso_grp,
+                 if_exists: true,
+                 algorithm: :concurrently
+
+    dup_ids = Rep.group(:representative_id)
+                 .having("COUNT(*) > 1")
+                 .pluck(:representative_id)
+    Rep.where(representative_id: dup_ids).delete_all if dup_ids.any?
```

Comment:
@patrick-brown-oddball I just want to make sure that you intend to delete ALL, not just the duplicates.
```suggestion
  dup_ids.each do |rep_id|
    # Keep the most recent, delete the rest
    reps = Rep.where(representative_id: rep_id).order(updated_at: :desc)
    reps.offset(1).delete_all
  end

```